public/app/core/components/LocalStorageValueProvider/LocalStorageValueProvider.tsx

```diff
@@ -24,8 +24,12 @@ export class LocalStorageValueProvider<T> extends PureComponent<Props<T>, State<
 
   onSaveToStore = (value: T) => {
     const { storageKey } = this.props;
-    store.setObject(storageKey, value);
-    this.setState({ value });
+    try {
+      store.setObject(storageKey, value);
+      this.setState({ value });
```

Comment:
Nitpick but the setState should probably be outside of the try catch. Not sure what can setState throw but if it throws it was not caught before.

Reply:
Fixed in https://github.com/grafana/grafana/pull/23650/commits/ae118f02eb4c579f9d046069e1156d1d0903b7b0